### ISSUE TITLE
about the training and testing data

### ISSUE THREAD

**Issue by piggy2008** [LintaoPeng/U-shape_Transformer_for_Underwater_Image_Enhancement#2]
Can you provide your image lists in the training and testing stage? The samples in the test folder are not all of the testing data.

**Comment by LintaoPeng** [comment]
Enhanced training set and test set data have been specially provided for you




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "LintaoPeng/U-shape_Transformer_for_Underwater_Image_Enhancement"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月15日(星期二) 中午1:34
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[LintaoPeng/U-shape_Transformer_for_Underwater_Image_Enhancement] about the training and testing data (Issue #2)





 
Can you provide your image lists in the training and testing stage? The samples in the test folder are not all of the testing data.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
